### DISCUSSION TITLE
Which browsers should we support?

### DISCUSSION THREAD

**Discussion by ecurrencyhodler** [saving-satoshi/saving-satoshi#216] [General]
As we prepare for the demo launch and strive to hit our deadline, we should discuss which browsers we want to focus on supporting.

So far, we have tested across Chrome, Firefox, and Safari. However there have been several edge cases with Safari that seem to be cropping up and blocking development. 

Two related issues are:

1. https://github.com/saving-satoshi/saving-satoshi/issues/213 which has delayed this PR from Shaavan https://github.com/saving-satoshi/saving-satoshi/pull/146#issuecomment-1441809114
2. This issue: https://github.com/saving-satoshi/saving-satoshi/issues/214

Historically Safari is not a great browser to include as it typically contains several bugs and isn't well supported. This is an issue I faced working at previous companies which ultimately led to them not supporting it. 

The pros is that this will decrease the scope and make it easier for us to hit our demo deadline. The cons are that it renders a poor experience for people who use safari.

What does everyone else think? Which browsers should we support? Is it worth supporting Safari?

**Comment by adamjonas** [comment]
Skip safari for now. We'll get browser data from our learners and adjust accordingly.
